tests/run/C_argc_argv-to-java_args/src/main/scala/CArgcArgvToJavaArgs.scala

```diff
@@ -0,0 +1,10 @@
+object CArgcArgvToJavaArgs {
+  def main (args: Array[String]) {
+    val len=args.length
+
+    assert(len == 3)
+    assert(args(0).equals ("hello"))

```

Comment:
We don't normally put space after function in function calls, i.e. `args(0).equals("hello")`